public/app/features/alerting/unified/state/actions.ts

```diff
@@ -701,6 +704,31 @@ interface UpdateNamespaceAndGroupOptions {
   newGroupName: string;
   groupInterval?: string;
 }
+// This method was introduced for making betterer happy (it was complaining about using as StoreState)
+const isStoreState = (something: unknown): something is StoreState => {
```

Comment:
You can try to provide some types to the `createAsyncThunk` function to get proper types for `getState` 
Ref: https://redux-toolkit.js.org/usage/usage-with-typescript#createasyncthunk